### DISCUSSION TITLE
Use of QurvE for substrate consumption and product formation rates

### DISCUSSION THREAD

**Discussion by LucasGCespedes** [NicWir/QurvE#16] [General]
Hi all,

During a discussion in my graduate class, we wondered if anyone has used QurvE to determine substrate consumption and product formation rates. Would it be possible to enter substrate or product concentration in data input in QurvE, instead of calculating dX/dt (umax) we would calculate dS/dt (substrate consumption rate) or dP/dt (product formation rate) in non-parametric fit. Does this make mathematical/scientific sense?

Best,

**Comment by NicWir** [comment]
Hi Lucas,

I have used QurvE for exactly that purpose. If divided by the biomass
concentration at the respective timepoints (if available), one can get a
time course of the differential biomass-specific product formation rates
(qp) and substrate consumption rates (*qs*) this way. This is
especially useful if *qs* and *qp* change during the course of a
fermentation process or if multiple substrates are present in the growth
medium.

However, if only one C substrate compound is provided, the biomass yield
*Yxs* and *qs* are best calculated as follows:
Create a biomass-against-substrate plot. Perform linear regressions on the
data points. The slope is *Yxs*. *qs* is then calculated by dividing the
growth rate μ by *Yxs*

Similarly, metabolite secretion rates (*qp*) are obtained by multiplying
*qs* with *Yps* (obtained as the slope from metabolite-against-substrate
plots).

These approaches are more robust and work well with single-substrate
cultivations and product formation that is strictly growth coupled. If
these conditions are not fulfilled, the approach you suggested can help
analysing the fermentation data in a more time-resolved manner.

I hope this helps. Please don't hesitate to reach out if you have any
further questions.

Best,

Nicolas
